artview/components/plot_radar.py

```diff
@@ -658,8 +615,8 @@ def _check_file_type(self):
                     self.rhi = True
             else:
                 self.rhi = True
-            #XXX this has only one effect: self._set_default_limits()
-            self._set_fig_ax_rhi()
+        self._set_default_limits()
+        self._set_fig_ax()
```

Comment:
open a PPI file and them a RHI, I believe the re-execution of _set_fig_ax is causing the problem